jekyll/_includes/global-nav.html

```diff
@@ -59,5 +59,12 @@
         </div>
       </nav>
     </div>
+
+    <!-- Experiment:   // https://app.optimizely.com/v2/projects/16812830475/experiments/20616121296/variations -->
+    <div class="dd-global-nav--links" style="display: none">
```

Comment:
Any reason why you used `--`?

Reply:
The styling methodology is a bit scattered, but it somewhat follows a bem-esque architecture.